lib/migrate/extractTextPlugin/extractTextPlugin.js

```diff
@@ -1,5 +1,15 @@
 const utils = require("../../utils/ast-utils");
 
+/**
+ *
+ * Check whether `node` is the invocation of the plugin denoted by `pluginName`
+ *
+ * @param {Object} j - jscodeshift top-level import
+ * @param {Node} node - ast node to check
+ * @param {String} pluginName - name of the plugin
+ * @returns {Boolean}
```

Comment:
Could you give a bit of description what the boolean indicates in the `@returns` ?